starter_code/index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title></title>
+  </head>
+  <body>
+    <div class="hero">
+      <h2>Relaxr</h2>
+      <h2>Get piece of mind with a single tap</h2>
+      <form>
+        <button type="button" name="button">Get it Now</button>
+    </div>
+
+    <div class="benefits">
+      <h3>Benefits</h3>
+      <p>
+        The perfect personal assistant. Relaxe does your job for you so you can enjoy life as it is meant to be.
+      </p>
+      <ul>
+        <li>Schedule meetings for you</li>
+        <li>Excel automation to complete your work for you</li>
+        <li>Responds to emails on your behalf</li>
+        <li>Does all your work for you with our revolutionary AI technology</li>
+      </ul>
+    </div>
+
+    <div class="bottom">
+      <p>"Relaxr changed my life. I've been able to travel the world, spend limited time working and my boss keeps thanking me for crushing work."></p>
+      <p>
+        Amanda, Intuit
+      </p>
+      <button type="button" name="button">Sign Up Now!</button>
+      <h2>Relaxr</h2>
+      <img src="starter_code/images/twitter.png" alt="twitter"/>
```

Comment:
The intuition here is for users to click the icon in the footer, so these images should be wrapped in `<a>` tags